### PR TITLE
Merge sky/yet-another-id-int

### PR DESCRIPTION
**Pull request by BenChand**
seismic-resource/master is already using these commits

### CODE REVIEW

**Review by skairunner — APPROVED** [review]

File: cognite/seismic/protos/v1/seismic_service_datatypes.proto [hunk 1]
```diff
@@ -76,6 +76,8 @@ message SeismicStore {
     File ingested_file = 6; // The info for the original file, if it was ingested as a file
     VolumeDef inline_volume_def = 7; // Volume definition for the store, indexed on inlines. Maps from an inline to all of its valid crosslines
     VolumeDef crossline_volume_def = 8; // Volume definition for the store, indexed on crosslines. Maps from a crossline to all of its valid inlines
+    TextHeader text_header = 9;
```
Comment (amorken) [inline cognite/seismic/protos/v1/seismic_service_datatypes.proto:79]: ...why is coverage 12? (or, why is there an "empty space" between tags 8 and 12 here?)

Comment (amorken) [reply cognite/seismic/protos/v1/seismic_service_datatypes.proto:79]: (reusing protobuf tags is ... not great.)

Comment (BenChand) [reply cognite/seismic/protos/v1/seismic_service_datatypes.proto:79]: There doesn't seem to be an obvious reason for it, I'll commit a fix to have coverage as 11

File: cognite/seismic/protos/v1/seismic_service_datatypes.proto [hunk 2]
```diff
@@ -78,7 +78,7 @@ message SeismicStore {
     VolumeDef crossline_volume_def = 8; // Volume definition for the store, indexed on crosslines. Maps from a crossline to all of its valid inlines
     TextHeader text_header = 9;
     BinaryHeader binary_header = 10;
-    Geometry coverage = 12;
+    Geometry coverage = 11;
```
Comment (amorken) [inline cognite/seismic/protos/v1/seismic_service_datatypes.proto:81]: Please don't change the tag of an existing protobuf field!

Comment (BenChand) [reply cognite/seismic/protos/v1/seismic_service_datatypes.proto:81]: Oh, my bad!

File: cognite/seismic/protos/v1/seismic_service_datatypes.proto (hunk 1 above)
Comment (amorken) [reply cognite/seismic/protos/v1/seismic_service_datatypes.proto:79]: I'm sorry I was unclear. The thing I was worried about here was that we would be reusing some tags that had been added at one point and then removed. It shouldn't be a big problem for this protobuf, though, as it has seen very limited use, basically just us and our SDK, so I'm not _very_ worried, but as a rule we should try to keep these things in mind when changing protobuf definitions: https://developers.google.com/protocol-buffers/docs/proto3#updating

Comment (amorken) [reply cognite/seismic/protos/v1/seismic_service_datatypes.proto:79]: Alright, I've gone spelunking in git history, and my working hypothesis is that @skairunner just copypasted the `Geometry coverage = 12` line from `message Seismic` above. Nevermind me, carry on. :p

**Review by amorken — APPROVED** [review]

**Pull request merged**